HopsanCore/include/ComponentUtilities/CSVParser.h

```diff
@@ -50,7 +50,7 @@ namespace hopsan {
 class HOPSANCORE_DLLAPI CSVParserNG
 {
 public:
-    CSVParserNG(const char separator_char = ',');
+    CSVParserNG(const char separator_char = ',', size_t linesToSkip=0);
```

Comment:
Maybe a parameter should be added to the CsvParser components (lookup table components) to allow the user to select the number of lines to skip.
There is also support for "comment char" typically # to autoskip any header lines (lines at the top only) starting with that char, maybe add support for that as well....

Reply:
I agree, but that belongs to a separate PR. Lookup tables component for 2D and 3D do not even allow choosing separator character yet.

Reply:
hmm I made those I think, why did I not add that, cant remember.... Yes we do it in a different PR
I can look it over once I have my new updated indexingcsvparser ready